src/hiccup/compiler.clj

```diff
@@ -22,11 +23,30 @@
 (defn- end-tag []
   (if (xml-mode?) " />" ">"))
 
+(defn iterate!
+  [callback coll]
```

Comment:
It looks like `iterate!` isn't used in a macro, unless I've missed some code somewhere. If that's right, it could be a private function. For consistency with the rest of the codebase, also put the arguments on the same line as the function name, i.e.

```clojure
(defn- iterate! [f coll]
```

Reply:
e4d8fa743281b800d8b0f4b2890e69b80be5283e
431e741c7c213b3f459c2bb48935566e37ce7ba8